functional_tests.go

```diff
@@ -4175,56 +4791,57 @@ func testFunctionalV2() {
 	}
 	resp, err = httpClient.Do(req)
 	if err != nil {
-		logger().Fatal("Error: ", err)
+		failureLog(function, args, startTime, "", "HTTP request to PresignedPutObject URL failed", err).Fatal()
 	}
 
 	newReader, err = c.GetObject(bucketName, objectName+"-presigned")
 	if err != nil {
-		logger().Fatal("Error: ", err)
+		failureLog(function, args, startTime, "", "GetObject failed", err).Fatal()
 	}
 
 	newReadBytes, err = ioutil.ReadAll(newReader)
 	if err != nil {
-		logger().Fatal("Error: ", err)
+		failureLog(function, args, startTime, "", "ReadAll failed", err).Fatal()
 	}
 
 	if !bytes.Equal(newReadBytes, buf) {
-		logger().Fatal("Error: bytes mismatch.")
+		failureLog(function, args, startTime, "", "Bytes mismatch", err).Fatal()
 	}
 
 	err = c.RemoveObject(bucketName, objectName)
 	if err != nil {
-		logger().Fatal("Error: ", err)
+		failureLog(function, args, startTime, "", "RemoveObject failed", err).Fatal()
 	}
 	err = c.RemoveObject(bucketName, objectName+"-f")
 	if err != nil {
-		logger().Fatal("Error: ", err)
+		failureLog(function, args, startTime, "", "RemoveObject failed", err).Fatal()
 	}
 	err = c.RemoveObject(bucketName, objectName+"-nolength")
 	if err != nil {
-		logger().Fatal("Error: ", err)
+		failureLog(function, args, startTime, "", "RemoveObject failed", err).Fatal()
 	}
 	err = c.RemoveObject(bucketName, objectName+"-presigned")
 	if err != nil {
-		logger().Fatal("Error: ", err)
+		failureLog(function, args, startTime, "", "RemoveObject failed", err).Fatal()
 	}
 	err = c.RemoveBucket(bucketName)
 	if err != nil {
-		logger().Fatal("Error:", err)
+		failureLog(function, args, startTime, "", "RemoveBucket failed", err).Fatal()
 	}
 	err = c.RemoveBucket(bucketName)
 	if err == nil {
-		logger().Fatal("Error:")
+		failureLog(function, args, startTime, "", "RemoveBucket should fail as bucket does not exist", err).Fatal()
```

Comment:
No point of logging nill error message, err:
failureLog(function, args, startTime, "", "RemoveBucket should fail as bucket does not exist"~~, err~~).Fatal()